### PR TITLE
12.0 mig l10n it abicab

### PR DESCRIPTION
**Pull request by primes2h**
Standard migration
- \_\_manifest\_\_.py updated 
- README.rst revamped

### CODE REVIEW

File: l10n_it_abicab/__manifest__.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+    'name': 'Italian localization - Codici bancari ABI/CAB',
+    'version': '12.0.1.0.0',
     'category': 'Localization/Italy',
+    'development_status': 'Production/Stable',
+    'summary': 'Base Bank ABI/CAB codes',
     'author': "Odoo Italia Network, Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_abicab',
     'license': 'AGPL-3',
     'depends': ['account'],
     'website': 'http://www.odoo-community.org',
```
Comment (eLBati) [inline l10n_it_abicab/__manifest__.py]: @primes2h duplicated `website`

File: l10n_it_abicab/models/abicab.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    abi = fields.Char(size=5, string='ABI')
+    cab = fields.Char(size=5, string='CAB')
+
+
+class ResPartnerBank(models.Model):
+
+    _inherit = "res.partner.bank"
+
+    bank_abi = fields.Char(size=5, string='ABI')
+    bank_cab = fields.Char(size=5, string='CAB')
```
Comment (eLBati) [inline l10n_it_abicab/models/abicab.py]: In customer form > accounting > bank accounts
when you create a bank account, selecting a bank with ABI and CAB filled, ABI and CAB of the account are not filled.

What do you think about making these 2 fields related?

Comment (primes2h) [reply l10n_it_abicab/models/abicab.py]: I don't get it.
Steps:
Accounting→Configuration→Bank Accounts→Create
Fill in "Bank Account Name" then select "Create and Edit..." in "Bank Account" field.
In the new form select a bank that has ABI and CAB set.
After that ABI and CAB fields below "Bank" get filled in the form.

File: l10n_it_abicab/__manifest__.py (hunk 1 above)
Comment (primes2h) [reply l10n_it_abicab/__manifest__.py]: Ops, fixed.

File: l10n_it_abicab/models/abicab.py (hunk 2 above)
Comment (primes2h) [reply l10n_it_abicab/models/abicab.py]: Ooops, sorry... :sweat_smile: :stuck_out_tongue_closed_eyes: 

Done!

File: l10n_it_abicab/models/abicab.py [hunk 3]
```diff
@@ -17,8 +18,8 @@ class ResPartnerBank(models.Model):
 
     _inherit = "res.partner.bank"
 
-    bank_abi = fields.Char(size=5, string='ABI')
-    bank_cab = fields.Char(size=5, string='CAB')
+    bank_abi = fields.Char(size=5, string='ABI', related='bank_id.abi')
+    bank_cab = fields.Char(size=5, string='CAB', related='bank_id.cab')
```
Comment (eLBati) [inline l10n_it_abicab/models/abicab.py]: @primes2h since they were DB columns, I would add `store=True`. What do you think?

Comment (primes2h) [reply l10n_it_abicab/models/abicab.py]: Better :+1: 

Done.

File: l10n_it_abicab/models/abicab.py [hunk 4]
```diff
@@ -17,8 +18,8 @@ class ResPartnerBank(models.Model):
 
     _inherit = "res.partner.bank"
 
-    bank_abi = fields.Char(size=5, string='ABI')
-    bank_cab = fields.Char(size=5, string='CAB')
+    bank_abi = fields.Char(size=5, string='ABI', related='bank_id.abi', store=True)
+    bank_cab = fields.Char(size=5, string='CAB', related='bank_id.cab', store=True)
```
Comment (eLBati) [inline l10n_it_abicab/models/abicab.py]: You exceeded 80 chars PEP8 limit

Comment (primes2h) [reply l10n_it_abicab/models/abicab.py]: Uff... sorry :disappointed:

**Pull request merged**